packages/services/api/src/modules/rate-limit/resolvers.generated.ts

```diff
@@ -1,7 +1,7 @@
 /* This file was automatically generated. DO NOT UPDATE MANUALLY. */
 import type { Resolvers } from './../../__generated__/types.next';
-import { Organization } from './resolvers/Organization';
+import { Organization as rate_limit_Organization } from './resolvers/Organization';
```

Comment:
This rename is expected because other modules are extending `type Organization`, so server preset adds module name to distinguish them, even if they are not all in one file